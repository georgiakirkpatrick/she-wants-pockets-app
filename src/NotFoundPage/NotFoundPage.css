.NotFoundPage {
    height: 80vh;
}