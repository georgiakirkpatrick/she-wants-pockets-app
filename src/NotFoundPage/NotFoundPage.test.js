import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import NotFoundPage from './NotFoundPage';

describe('NotFoundPage', () => {
        it ('renders without crashing', () => {
            const div = document.createElement('div');
            ReactDOM.render(<NotFoundPage />, div);
            ReactDOM.unmountComponentAtNode(div);
        });
    
        it ('render the UI as expected', () => {
            const tree = renderer
                .create(<NotFoundPage />)
                .toJSON();
            expect(tree).toMatchSnapshot()
        });
    });