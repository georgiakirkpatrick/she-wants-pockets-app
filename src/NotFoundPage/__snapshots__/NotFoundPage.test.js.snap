// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage render the UI as expected 1`] = `
<section
  className="NotFoundPage"
>
  <header>
    <h1>
      That page does not exist.
    </h1>
  </header>
</section>
`;
